demo/demo.py

```diff
@@ -132,6 +132,9 @@ def __exit__(self, type, value, traceback):
                 sounds_path=os.path.join(DEMO_DIR, "short_noises"),
                 min_snr_in_db=0,
                 max_snr_in_db=8,
+                min_absolute_noise_rms_db=-30,
+                max_absolute_noise_rms_db=-10,
+                noise_power="absolute",
```

Comment:
Maybe it's better to have two separate configs of AddShortNoises in the demo script, one with noise_power="relative" and one with noise_power="absolute"? Then we can listen to both. I also did this with ApplyImpulseResponse a few lines above, and I gave the second one a different `"name"` 😃

Reply:
Yes, good idea ! I'll change it :)